flutter-studio/src/io/flutter/project/FlutterSettingsStep.form

```diff
@@ -23,9 +23,9 @@
           <grid row="12" column="0" row-span="1" col-span="1" vsize-policy="6" hsize-policy="1" anchor="0" fill="2" indent="0" use-parent-layout="false"/>
         </constraints>
       </vspacer>
-      <component id="3d5fb" class="javax.swing.JLabel">
+      <component id="3d5fb" class="javax.swing.JLabel" binding="myLanguageLabel">
         <constraints>
-          <grid row="9" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+          <grid row="9" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="1" use-parent-layout="false"/>
```

Comment:
Just confirming that the indentation increase here is intended -

Reply:
Not intended, and it really did need to be restored, thanks.